### ISSUE TITLE
Animated Avatars Not Showing (again?)

### ISSUE THREAD

**Issue by GENKEJ** [SevenTV/Extension#936]
I've read through an older thread explaining exactly the same issue with the answer that it should be fixed within a few days.
Not sure if it just started happening or if it's been like this for some time, but i want to make sure this isn't a problem on my end since i've tried multiple ways with no success.
![Snímek obrazovky 2023-11-16 194930](https://github.com/SevenTV/Extension/assets/151072364/0caac318-058e-41e6-9975-b77301794ad6)

**Comment by AnatoleAM** [comment]
Yes, we're currently facing some performance issues heightened by animated avatars. We're working on it and hope to have a fix out sometime next week
